Assets/Script/Menu/ProfileList/ProfileListMenu.cs

```diff
@@ -94,12 +94,27 @@ private void AddListGroup(string header, IEnumerable<YargProfile> profiles)
                 _navigationGroup.AddNavigatable(go);
             }
         }
+        
+        private string GetUniqueProfileName(string profileName)
```

Comment:
This should be made static

```suggestion
        private static string GetUniqueProfileName(string profileName)
```